setup.py

```diff
@@ -55,6 +55,7 @@
         "teachable": ["chromadb"],
         "lmm": ["replicate", "pillow"],
         "graphs": ["networkx~=3.2.1", "matplotlib~=3.8.1"],
+        "redis": ["redis"],
```

Comment:
Optional: update Installation.md about this extra option.

Reply:
Done.